fs-repo-12-to-13/vendor/github.com/ipfs/fs-repo-migrations/tools/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
AFAIK we are not uploading vendor folders.

Reply:
We do for this repo:
```console
$ find -type d -name vendor | sort --version-sort 
./fs-repo-0-to-1/vendor
./fs-repo-1-to-2/vendor
./fs-repo-2-to-3/vendor
./fs-repo-3-to-4/vendor
./fs-repo-4-to-5/vendor
./fs-repo-5-to-6/vendor
./fs-repo-6-to-7/vendor
./fs-repo-7-to-8/vendor
./fs-repo-8-to-9/vendor
./fs-repo-9-to-10/vendor
./fs-repo-10-to-11/vendor
./fs-repo-11-to-12/vendor
./fs-repo-12-to-13/vendor
```